sklearn/ensemble/tests/test_voting.py

```diff
@@ -503,12 +513,15 @@ def test_transform():
 )
 @pytest.mark.parametrize("drop", [None, 'drop'])
 def test_none_estimator_with_weights(X, y, voter, drop):
-    # check that an estimator can be set to None and passing some weight
+    # check that an estimator can be set to 'drop' and passing some weight
```

Comment:
test name should be updated :p 

also mark as todo remove parametrization maybe